docs/labs.md

```diff
@@ -120,6 +120,15 @@ This feature allows users to place and join native [MSC3401](https://github.com/
 
 If you're enabling this at the deployment level, you may also want to reference the docs for the `element_call` config section.
 
+## Disable per-sender encryption for Element Call (`feature_disable_call_per_sender_encryption`)
+
+The default for embedded Element Call in Element Web is per participant sender key encryption. 
```

Comment:
Nit:

```suggestion
The default for embedded Element Call in Element Web is per-participant sender key encryption. 
```

Reply:
... but only in encrypted rooms? https://github.com/matrix-org/matrix-react-sdk/pull/11837/files#diff-aa8e05e46de65e6a368e1898db9dd420a41e7001bd470febc5212704add0bdeeR664

Reply:
Also, what is "sender key encryption"?

Reply:
> Also, what is "sender key encryption"?

Each sender has its own different key. I think you are right that we can just go with per-participant encryption.